src/main/java/Todos.java

```diff
@@ -0,0 +1,29 @@
+public class Todos {
```

Comment:
Perhaps you can name this as Todo instead of Todos to be consistent (with Deadline and Event)